js/crashReporter.js

```diff
@@ -0,0 +1,75 @@
+const { crashReporter } = require('electron');
+const { getMultipleConfigField } = require('./config.js');
+
+const log = require('./log.js');
+const logLevels = require('./enums/logLevels.js');
+
+/**
+ * Method that returns all the required field for crash reporter
+ *
+ * @param extras {object}
+ * @return {Promise<any>}
+ */
+function getCrashReporterConfig(extras) {
+    return new Promise((resolve, reject) => {
+        getMultipleConfigField(['url', 'crashReporter'])
```

Comment:
we can probably do this once and store the data in a variable rather than calling the `getMultipleConfigField` every time.